doc/index.rst

```diff
@@ -2,12 +2,16 @@ xarray: N-D labeled arrays and datasets in Python
 =================================================
 
 **xarray** (formerly **xray**) is an open source project and Python package
-that aims to bring the labeled data power of pandas_ to the physical sciences,
-by providing N-dimensional variants of the core pandas data structures.
-
-Our goal is to provide a pandas-like and pandas-compatible toolkit for
-analytics on multi-dimensional arrays, rather than the tabular data for which
-pandas excels. Our approach adopts the `Common Data Model`_ for self-
+that aims to make working with labelled multi-dimensional arrays simple,
+efficient, and fun!
+
+Labelled multi-dimensional (a.k.a. N-dimensional) arrays are encountered in
+many fields, especially physical sciences, engineering, and finance.
+But multi-dimensional data doesn't fit neatly into pandas_, python's most
```

Comment:
If we are going to contrast directly with Pandas, I think we need to say what Pandas is first. Maybe also provide an example of what Pandas does (tabular data structures).

Reply:
I agree that we can't assume that readers know what Pandas is - I certainly didn't. I think that users coming from a more data science background will have used Pandas but those coming from a more low-level numpy/MATLAB/Fortran/C++ point-of-view